pkgs/leak_tracker_testing/README.md

```diff
@@ -1,5 +1,21 @@
 [![pub package](https://img.shields.io/pub/v/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing)
+[![package publisher](https://img.shields.io/pub/publisher/leak_tracker_testing.svg)](https://pub.dev/packages/leak_tracker_testing/publisher)
 
-Coming soon!
+Coming soon! See https://github.com/flutter/devtools/issues/3951.
 
-See full documentation at https://github.com/dart-lang/leak_tracker.
+The text below is under construction.
+
+## What is this?
+
+This is test helpers for [leak_tracker](https://pub.dev/packages/leak_tracker).
```

Comment:
Same comment as above; it would be good to answer the question about why this package is separate from `leak_tracker`.